File: tt/DS_practice/src/cheggoct/Driver.java
package cheggoct;

import java.util.Scanner;
/**
 * Driver class for running player
 * @author yourname
 *
 */
public class Driver {
/**
 * main method
 * @param args
 */
	public static void main(String[] args) {

		String fn,ln;
		int num,user_choice;
		Player players[] = new Player[11];
		int index=0;

		System.out.println("Welcome to Roster Editor!");
		Scanner sc = new Scanner(System.in);

		while(true)//running in loop
		{
			System.out.print("1) Add New Player\n2) View Player(s)\n3) Exit Program\n\n");
			System.out.print("Plese enter your selection : ");
			user_choice = sc.nextInt();
			//on selection choice
			if(user_choice==1)
			{
				if(index>=11)//if roaster fulls printing 
				{			//the roaster info and exiting the program
					System.out.println("Roaster full!");
					for(int i=0;i<index;i++)
					{
						System.out.println(players[i]);
					}
					System.out.println("*****************************\n\n");
					System.out.println("\nThanks for using our program - Goodbye!\n");
					break;

				}
				else{
					//creating new player and adding to array
					Player temp = new Player();
					System.out.print("Please enter a first name : ");
					fn=sc.next();
					System.out.print("Please enter a last name : ");
					ln=sc.next();
					System.out.print("Please enter number (1-99) : ");
					num=sc.nextInt();
					temp.setFirstName(fn);//setting playesrs property
					temp.setLastName(ln);
					temp.setNumber(num);
					players[index]=temp;
					System.out.print("**Player Created**\n\n");
					index++;
				}
			}
			else if (user_choice==2)
			{
				System.out.println("\n**IUPUI Football Roaster**");
				for(int i=0;i<index;i++)
				{
					System.out.println(players[i]);
				}
				System.out.println("*****************************\n\n");
			}
			else if(user_choice==3)
			{
				System.out.println("\nThanks for using our program - Goodbye!\n");
				break;
			}
			else{
				System.out.println("Invalid choice! Please select option #1, #2, or #3.\n");
			}
		}
		sc.close();
	}

}

File: tt/c_cppCode/cheggoct/Driver.cpp
//For running and executing the program
//Driver.cpp
#include<string>
#include<iostream>
#include "Player.cpp"//imported player

using namespace std;
/*Main function*/
int main()
{
	//Roaster of max 11
	player *roaster[11];
	int count=0;
	string fn,ln;
	int nu,choice;
	
	cout<<"Welcome to Roster Editor!\n";
	//displaying menu for adding , viewing roaster
	while(1)
	{
		cout<<"1) Add New Player\n2) View Player(s)\n3) Exit Program\n";
		cout<<"Plese enter your selection : ";
		cin>>choice;//asking for user choice
		if(choice==1)//based on choice performing the add/view/exit
		{
			if(count>=11)//once the roaster  get full
			{			//printing all player info and terminating the program
			
				cout<<"Roaster full!!\n\n";
				cout<<"\n**IUPUI Football Roaster**"<<endl;
				for(int i=0;i<count;i++)
				{
				cout<<roaster[i]->getNumber()<<") "<<roaster[i]->getFirstName()<<" "<<roaster[i]->getLastName()<<endl;
				}
				cout<<"*****************************\n\n";
				cout<<"\nThanks for using our program - Goodbye!\n";
				exit(0);
			}
			else{
			
			cout<<endl;
			cout<<"Please enter a first name : ";
			cin>>fn;
			cout<<"Please enter a last name : ";
			cin>>ln;
			cout<<"Please enter number (1-99) : ";
			cin>>nu;
			//using index number of player array
			//and creating dynamically 
			roaster[count]=new player();
			roaster[count]->setFirstName(fn);
			roaster[count]->setLastName(ln);
			roaster[count]->setNumber(nu);
			cout<<"**Player Created**\n\n";
			count++;
		}
		}
		else if (choice==2)
		{
			cout<<"\n**IUPUI Football Roaster**"<<endl;
			for(int i=0;i<count;i++)
			{
				cout<<roaster[i]->getNumber()<<") "<<roaster[i]->getFirstName()<<" "<<roaster[i]->getLastName()<<endl;
			}
			cout<<"*****************************\n\n";
		}
		else if(choice==3)
		{//freeing the memory
			for(int i=0;i<count;i++) 
				delete roaster[i];
			cout<<"\nThanks for using our program - Goodbye!\n";
			exit(0);
		}
		else{
			cout<<"Invalid choice! Please select option #1, #2, or #3.\n"<<endl;
		}
	}
	return 0;
}
